values.yaml

```diff
@@ -254,6 +261,41 @@ server:
     # https_proxy: http://localhost:3128,
     # no_proxy: internal.domain.com
 
+
+# Add configuration for Consul servers running externally,
+# i.e. outside of Kubernetes.
+# This information is required if Consul servers are running
+# outside of k8s and you’re setting global.tls.enableAutoEncrypt to true.
+externalServer:
```

Comment:
I know we usually go singular but this reads weird to me. Is it referring to having a single external server? I think this should be `externalServers`

Reply:
It's referring to any number of servers; it could be 1 or more. I originally proposed plural in the RFC, but changed it to singular because of other values. Thinking about this more though, these values seem to refer to "features" and in that sense, the feature here would be `externalServers`.

Reply:
👍 I think `externalServers` makes sense then.